src/ethereum_clis/types.py

```diff
@@ -93,3 +101,19 @@ class TransitionToolOutput(CamelModel):
     alloc: Alloc
     result: Result
     body: Bytes | None = None
+
+
+class TransitionToolState(CamelModel):
+    """Transition tool state."""
```

Comment:
the naming here looks misleading. because the first thing I think of is ethereum state

Reply:
agreed

Reply:
Changed to TransitionToolContext in latest commit 👍